Python Engineering Intern/README.md

```diff
@@ -58,3 +58,11 @@ Cite any external piece of code you're using, preferably with a link to the sour
 2. [Data Extraction from Unstructured PDFs](https://www.analyticsvidhya.com/blog/2021/06/data-extraction-from-unstructured-pdfs/)
 3. [Pandas](https://pandas.pydata.org/docs/user_guide/index.html)
 4. [Version control with git](https://try.github.io/)
+
+## Approach to the problem statement
+1. I used PyPDF2 and camelot to convert the pdf table to csv file.
```

Comment:
@Prajwal-Adiga-15 I think you forgot to add the script to convert the pdf table to csv file. Can you kindly add that and then we can continue on the code review?

Reply:
@debdutgoswami Sorry! for the inconvenience, I have added the  pdf1_to_csv.py and pdf2_to_csv.py files for the conversion of the pdf table to csv file.